exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -68,12 +74,9 @@
   }
 
   @Test
-  void createTestKitForSingleService() {
-    TestService service;
-    try (TestKit testKit = TestKit.forService(TestServiceModule.class)) {
-      service = testKit.getService(TestService.SERVICE_ID, TestService.class);
-      checkTestServiceInitialization(testKit, service);
-    }
+  void createTestKitForSingleService(TestKit testKit) {
```

Comment:
It is nice if we can inject Testkit directly, but the present implementation makes it impossible to configure the testkit differently for different tests (without `@Nested` or separating the tests into multiple files).

Reply:
Yes, I thought about that, but is it a frequent case for users to test their service with different TestKit configuration? It seems that usually they'd use the same configuration in the same test suite or use `@Nested`. Even if they need that, they could just not inject it and create it manually. If not, what do you propose?

Reply:
auditor/validator? 

> If not, what do you propose?

Please see in jira.